### PR TITLE
DOCS-11126: Improvements to Installation Tutorials

### PR DESCRIPTION
**Pull request by rkumar-mongo**
- All tutorials include 'Verify MongoDB started successfully' step
- All tutorials ensure user attempts to connect successfully to the mongod

---
This change is [Reviewable](https://reviewable.io/reviews/mongodb/docs/3169)

### CODE REVIEW

**Comment by rkumar-mongo** [comment]
@kay-kim reviewed by Andrew and Shannon, please merge/pub at your earliest convenience

File: source/includes/steps-configure-windows-service-for-mongodb.yaml [hunk 1]
```diff
@@ -46,8 +46,39 @@ inherit:
...
 ---
-title: Stop or remove the MongoDB service as needed.
 stepnum: 6
+source:
+  file: steps-run-mongodb-on-linux.yaml
+  ref: verify
+pre: |
+  Verify that MongoDB has started successfully by checking the 
+  log file at ``c:\data\log\mongod.log`` for the following line:
+post: |
```
Comment (kay-kim) [inline source/includes/steps-configure-windows-service-for-mongodb.yaml]: what line?

File: source/includes/steps-create-manually-windows-service-for-mongodb.yaml [hunk 2]
```diff
@@ -64,8 +64,39 @@ action:
...
 ---
-title: Stop or remove the MongoDB service as needed.
 stepnum: 6
+source:
+  file: steps-run-mongodb-on-linux.yaml
+  ref: verify
+pre: |
+  Verify that MongoDB has started successfully by checking the 
+  log file at ``c:\data\log\mongod.log`` for the following line:
+post: |
```
Comment (kay-kim) [inline source/includes/steps-create-manually-windows-service-for-mongodb.yaml]: ? what line?

File: source/includes/steps-configure-windows-service-for-mongodb.yaml (hunk 1 above)
Comment (rkumar-mongo) [reply source/includes/steps-configure-windows-service-for-mongodb.yaml]: I think this pulls in the `action` from the inherited file, so the line ends up being the `initAndListen` log file. Staged here: https://docs-mongodbcom-staging.corp.mongodb.com/rkumar/DOCS-11126/tutorial/install-mongodb-on-windows.html#verify-that-mongodb-has-started-successfully

**Pull request merged**